libraries/AP_NavEKF3/AP_NavEKF3_core.h

```diff
@@ -569,10 +569,16 @@ class NavEKF3_core : public NavEKF_core_common
         float           delTime;    // time interval that the measurement was accumulated over (sec)
     };
         
+    // Specifies the rotation order used for the Tait-Bryan or Euler angles where alternative rotation orders are available
+    enum class rotationOrder {
```

Comment:
In my mind at least, it is very tempting to align this rotationOrder enum with the "type" we log into the DAL.  Perhaps that's better left to a separate PR although doing it now would save on testing effort.

Reply:
Would prefer to do that separately as a 'non-functional' change request. The other issue is that such a change does change the log file definition.